iron-doc-api.html

```diff
@@ -70,21 +69,37 @@
       </h2>
 
       <template is="dom-repeat"
+                items="[[_public(descriptor.staticMethods)]]"
+                sort="_compareDescriptors">
+        <iron-doc-function descriptor="[[item]]" static
+                           anchor-id$="[[fragmentPrefix]]method-[[item.name]]">
```

Comment:
this anchor id should probably have `static` or similar in it somewhere, so that we can have static methods with the same name as instance methods